src/test/OptionSettlement.t.sol

```diff
@@ -79,363 +83,66 @@ contract OptionSettlementTest is DSTest, NFTreceiver {
     }
 
     function setUp() public {
-        dai = IERC20(DAI);
-        usdc = IERC20(USDC);
-        weth = IERC20(WETH);
-
         engine = new OptionSettlementEngine();
 
-        option = IOptionSettlementEngine.Option({
-            underlyingAsset: WETH,
-            exerciseAsset: DAI,
-            settlementSeed: 1234567,
-            underlyingAmount: 1 ether,
-            exerciseAmount: 3000 ether,
-            exerciseTimestamp: uint40(block.timestamp),
-            expiryTimestamp: (uint40(block.timestamp) + 604800)
-        });
+        IOptionSettlementEngine.Option memory option = IOptionSettlementEngine
```

Comment:
Having this obj in contract wide storage would help shortening other tests as we would not have to re-declare a whole option, just set a new one in memory to point to the storage one and modify the desired attributes